internal/common/common_test.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2018 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
+
+import (
+	"errors"
+	"testing"
+)
+
+func TestLookupBlockdev(t *testing.T) {
```

Comment:
Is it too unreliable to test a positive case both here and in `TestLookupVolName`? Is it safe to assume that `/dev/loop0` is available?